main.java

```diff
@@ -26,38 +26,44 @@ public static void main(String[]args)
         //initalize student   
         student.setStudent();
 
-        //Display menu and select an option 
-        displayMenu();
-        int input = in.nextInt();
-        
-        //If the input is not an option return an error and request a new response
-        while(input < 1 || input > 4) {
-            System.out.println("\nInput Error. Enter a Valid Integer!");
-            displayMenu();
-            input = in.nextInt();
-        }
+        //boolean var to loop back to login
+        boolean rerun = true;
 
-        //Show choice
-        switch (input) { 
-            //Show the users reservations
-            case 1: 
-                System.out.println("Reservations:"); 
-                Student.showReservations(); 
-                break;
-            //Displays the users funds
-            case 2: 
-                System.out.println("Funds: " + Student.getMoney());
+        while(rerun == true)        
+        {
+            //Display menu and select an option 
+            displayMenu();
+            int input = in.nextInt();
```

Comment:
If you make a reservation it crashes since this variable is initialized within the loop.